internal/docstore/drivertest/drivertest.go

```diff
@@ -976,6 +984,16 @@ func testMultipleActions(t *testing.T, coll *ds.Collection) {
 		{KeyField: "testMultipleActions11", "s": "k"},
 		{KeyField: "testMultipleActions12", "s": "l"},
 	}
+
+	actions := coll.Actions()
```

Comment:
Let's do the `forEach` + `Delete` way inside `defer` to clean up after the test docs like in the query and as tests.

Reply:
Done.